third_party/move/tools/move-cli/src/base/test.rs

```diff
@@ -207,7 +209,7 @@ pub fn run_move_unit_tests<W: Write + Send>(
     // control back to the Move package system.
     let (_, model_opt) = build_plan.compile_with_driver(
         writer,
-        &CompilerConfig::default(),
+        &build_config.compiler_config,
```

Comment:
@wrwg Should we use build_config.compiler_config or keep CompilerConfig::default() with compiler version specified in build_config?

Reply:
Not sure, you can check what they are doing different, sounds like what you are doing here is right.